parquet-column/src/test/java/org/apache/parquet/column/values/Utils.java

```diff
@@ -76,7 +77,7 @@ public static void writeData(ValuesWriter writer, String[] strings)
   public static int[] readInts(ValuesReader reader, byte[] data, int offset, int length)
       throws IOException {
     int[] ints = new int[length];
-    reader.initFromPage(length, data, offset);
```

Comment:
There are a number of places where I think we should make it consistent and avoid passing an offset. Right now this would mix a ton of new changes into the patch. This has been dragging on for a long time. I would prefer to make this a follow up task.